net/gce.sh

```diff
@@ -240,6 +242,11 @@ while getopts "h?p:Pn:c:r:z:gG:a:d:uxf" opt "${shortArgs[@]}"; do
     validatorMachineType=$bootstrapLeaderMachineType
     blockstreamerMachineType=$bootstrapLeaderMachineType
     ;;
+  M)
```

Comment:
Can you use a long option?  I want to stop using short ops because it's turned into alphabet soup.